sql/telemetry_derived/ssl_ratios_v1/metadata.yaml

```diff
@@ -0,0 +1,10 @@
+friendly_name: SSL Ratios
+description: >
+  Percentages of page loads Firefox users have performed that were conducted over SSL.
+owners:
+  - chutten@mozilla.com
+labels:
+  application: firefox
+  schedule: daily
+  public_json: true
```

Comment:
I'll update the existing script for publishing metadata to BigQuery in a separate PR so that key-value pairs with boolean values get published as tags.